nodejs/packages/layer/src/wrapper.ts

```diff
@@ -212,6 +226,44 @@ function createInstrumentations() {
   ];
 }
 
+function getPropagator(): TextMapPropagator {
```

Comment:
Consider adding some unit tests.

Reply:
@tylerbenson added tests, could you please check